assets/css/card.css

```diff
@@ -329,3 +311,183 @@
   font-size: 1.5rem;
   background-color: var(--white-shade);
 }
+
+@media only screen and (max-width: 480px) {
+  .Card-container {
+    width: 100%;
```

Comment:
Suggestion: Use variables for media query breakpoints, It will be easier to change and manage it.